bpf/lib/maps.h

```diff
@@ -85,28 +85,6 @@ struct bpf_elf_map __section_maps CONFIG_MAP = {
 };
 #endif
 
-#ifdef ENABLE_IPV4
-struct bpf_elf_map __section_maps PROXY4_MAP = {
```

Comment:
Todo: Search for all remaining references in the codebase. For example, proxy map sizes, map name definitions like this. At least needs updates to `pkg/datapath/linux/config.go`.

Reply:
TODO.

Reply:
Done (in the next update).